Sources/SourceKitLSP/SourceKitServer.swift

```diff
@@ -620,12 +620,7 @@ extension SourceKitServer: MessageHandler {
   private func handleImpl(_ params: some NotificationType, from clientID: ObjectIdentifier) async {
     let notification = Notification(params, clientID: clientID)
 
-    logger.log(
-      """
-      Received notification
-      \(notification.forLogging)
-      """
-    )
+    logger.log("Received notification: \(notification.forLogging)")
```

Comment:
Presumably you found this looks nicer without the newline?

Reply:
Yes, my use case for this was that I could grep for `Received notification` and see the types of all the notifications that got sent.